user-impl/src/main/java/com/example/auction/user/impl/PUserCommand.java

```diff
@@ -12,13 +13,33 @@
     final class CreatePUser implements PUserCommand, PersistentEntity.ReplyType<PUser> {
         private final String name;
         private final String email;
+        private final String passwordHash;
 
         @JsonCreator
         public CreatePUser(PUser user) {
```

Comment:
I think this constructor should also be deleted, and it can use the one generated by default from Lombox (which will take the name, email and password hash directly as parameters, rather than requiring a `PUser`).